config/generate/main.go

```diff
@@ -63,6 +63,11 @@ func main() {
 		"LogFieldsAttribute":        newFields(&meta.LogFieldsAttribute{}),
 	}
 
+	blockNamesMap := map[string]string{
+		"oauth2_ac":       "beta_oauth2",
```

Comment:
I get your point but the both oauth2 blocks seems to be not found atm. Left side navigation links do not match and we need to delete the old items from the search index if updated properly since the result is still oauth2reqauth.

Reply:
Maybe a caching issue. Seems to work now, at least for me. I've deleted the old search index items. Should be okay now.